desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.tsx

```diff
@@ -326,56 +318,45 @@ const StorageBrowserTable = ({
           }}
         />
         <div className="hue-storage-browser__actions-bar-right">
-          {viewType === BrowserViewType.dir && (
-            <>
-              <StorageBrowserActions
-                selectedFiles={selectedFiles}
-                setLoadingFiles={setLoadingFiles}
-                onSuccessfulAction={refetchData}
-              />
-              <Dropdown
-                overlayClassName="hue-storage-browser__actions-dropdown"
-                menu={{
-                  items: newActionsMenuItems,
-                  className: 'hue-storage-browser__action-menu'
-                }}
-                trigger={['hover', 'click']}
-              >
-                <PrimaryButton data-event={''}>
-                  {t('New')}
-                  <DropDownIcon />
-                </PrimaryButton>
-              </Dropdown>
-            </>
-          )}
+          <StorageBrowserActions
+            selectedFiles={selectedFiles}
+            setLoadingFiles={setLoadingFiles}
+            onSuccessfulAction={refetchData}
+          />
+          <Dropdown
+            overlayClassName="hue-storage-browser__actions-dropdown"
+            menu={{
+              items: newActionsMenuItems,
+              className: 'hue-storage-browser__action-menu'
+            }}
+            trigger={['hover', 'click']}
+          >
+            <PrimaryButton data-event={''}>
+              {t('New')}
+              <DropDownIcon />
+            </PrimaryButton>
+          </Dropdown>
         </div>
       </div>
 
       <Spin spinning={loadingFiles}>
-        {viewType === BrowserViewType.dir && (
-          <Table
-            className={className}
-            columns={getColumns(tableData[0] ?? [])}
-            dataSource={tableData}
-            onRow={onRowClicked}
-            pagination={false}
-            rowClassName={rowClassName}
-            rowKey={(record, index) => record.path + '' + index}
-            rowSelection={{
-              type: 'checkbox',
-              ...rowSelection
-            }}
-            scroll={{ y: tableHeight }}
-            data-testid={`${testId}`}
-            locale={locale}
-            {...restProps}
-          />
-        )}
-
-        {viewType === BrowserViewType.file && (
-          // TODO: code for file view
-          <div> File view</div>
-        )}
+        <Table
+          className={className}
+          columns={getColumns(tableData[0] ?? {})}
+          dataSource={tableData.slice(2) ?? []}
```

Comment:
The tableData will be an array always, either and empty one or non-empty one, although having nullish coalescing operator doesn't harm.

Reply:
I see, well then the unnecessary nullish coalescing operator only adds confusion (at least for me) and should probably be removed :-)

Reply:
Ok, reverting it.

Reply:
So given what we know about nullish coalescing and that tableData will always be an array, I think we should just write
`dataSource={tableData?.slice(2)}`

since slicing an empty array will return an empty array.

Reply:
Make sense, noted and changed.